adt_press/utils/languages.py

```diff
@@ -1,79 +1,47 @@
-LANGUAGE_MAP = {
```

Comment:
From everything I can tell this was basically here to map a language code to a language name, which pycountry is doing for us now.